packages/core/core/create-handler.js

```diff
@@ -34,6 +34,7 @@ const createHandler = (optionByName = {}) => {
             // Helps mongoose reuse the connection.  Lowers response times.
             context.callbackWaitsForEmptyEventLoop = false;
 
+            // todo: this should not be necessary anymore
```

Comment:
Kk. There are cases where we definitely don't want to, which I believe falls under.... a use case? an adapter?